tools/release/java/src/main/java/com/ibm/icu/dev/tools/docs/StableAPI.java

```diff
@@ -645,30 +655,31 @@ private void simplifyPrototype() {
                 return;
             }
             final String prototype0 = prototype;
-            for (int i = 0; i < simplifyList.length; i+= 2) {
-                prototype = prototype.replaceAll(simplifyList[i+0],simplifyList[i+1]);
+            for (int i = 0; i < simplifyList.length; i += 2) {
+                prototype = prototype.replaceAll(simplifyList[i + 0], simplifyList[i + 1]);
             }
-            if(!prototype0.equals(prototype)) {
+            if (!prototype0.equals(prototype)) {
                 addSimplification(prototype0, prototype);
             }
         }
-//        private Element toXml(Document doc){
-//            Element  ele = doc.createElement("func");
-//            ele.setAttribute("prototype", prototype);
-//            ele.setAttribute("id", id);
-//            ele.setAttribute("status", status);
-//            return ele;
-//        }
+        // private Element toXml(Document doc){
```

Comment:
Please either undo the reformatting of this block or delete it if it's obsolete.